# This message type is created for relative measurements.
time 			current_time
string 			parent_name
int16 			marker
geometry_msgs/Twist 	pose_parent
geometry_msgs/Twist 	twist_relative # Should become a pose!
